snapcraft/store/client.py

```diff
@@ -34,8 +34,6 @@
 from ._legacy_account import LegacyUbuntuOne
 from .onprem_client import ON_PREM_ENDPOINTS, OnPremClient
 
-_TESTING_ENV_PREFIXES = ["TRAVIS", "AUTOPKGTEST_TMP"]
```

Comment:
If you want to keep the user-agent change, you could detect the `CI` and/or [`PYTEST_CURRENT_TEST`](https://docs.pytest.org/en/latest/example/simple.html#pytest-current-test-environment-variable).

I'm not sure how valuable this is, but it is an option.

Reply:
Since no one is needing it, I'll omit it for simplicity.